cpu/sam0_common/periph/rtc_rtt.c

```diff
@@ -124,6 +124,7 @@ static void _poweroff(void)
 #endif
 }
 
+#if defined(MODULE_PERIPH_RTC) || defined(RTC_NUM_OF_TAMPERS)
```

Comment:
```suggestion
__attribute__((unused))
```

Would this work as well? I do not like preprocessor conditionals :)

@kfessel is working on a `MAYBE_UNUSED` define that (in addition to being more readable) would make it easy to add support for compilers other than GCC and clang (that both support `__attribute__((unused))` to indicate that the following function can be garbage collected without issuing a warning). Maybe it would make sense to wait for this to get merged, if it isn't already.

Reply:
isn't the issue with `__attribute__((unused))` that it cannot be used in every case (like, if it uses symbols that are not available)? Similar to `IS_USED()`, two ways of doing things.

Reply:
Note that this compiled with GCC and no preprocessor conditionals.

I think the reason is that GCC doesn't complain about unused static inline functions in any case, while clang only allows unused static inline functions in included header files.